Extension error building docs

### What went wrong?

I created a newly up-to-date devel environment this morning, but `make clean html` fails.

### Operating System

Linux

### Version

1.5.1.post377+g0fe89d5667

### Python Version

3.12.0

### Code to Reproduce

```python
# BASH commands
create -n devel
conda activate devel
conda config --env --add channels conda-forge --add channels conda-forge/label/testing
conda install --file ci/requirements.txt --file ci/extra_requirements.txt --file ci/test_requirements.txt
pip install -e .
conda install --file ci/doc_requirements.txt
cd docs
make clean html
```


### Errors, Traceback, and Logs

```pytb
embedding documentation hyperlinks...

Extension error (sphinx_gallery.docs_resolv):
Handler <function embed_code_links at 0x7fcad89b32e0> for event 'build-finished' threw an exception (exception: Expecting property name enclosed in double quotes: line 1 column 2 (char 1))
make: *** [html] Error 2
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -210,6 +210,7 @@ Then, still from within your ``devel`` environment,
 
 * Navigate to the docs folder ``cd docs``
 * Remove any old builds and build the current docs ``make clean html``
+  * (Try ``make cleanall html`` if ``make clean html`` fails)
 * Open ``docs/build/html/index.html`` and see your changes!
 
 ### `doc-server.py`

